src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1837,6 +1837,28 @@ static void uct_ib_mlx5_devx_check_xgvmi(uct_ib_mlx5_md_t *md, void *cap_2,
     }
 }
 
+static void uct_ib_mlx5_devx_check_multi_path(uct_ib_mlx5_md_t *md, void *cap,
+                                              void *cap_2, uct_ib_device_t *dev)
+{
+    md->super.multi_path.rc_rdma  = UCT_IB_MLX5DV_GET(cmd_hca_cap, cap,
+                                                      multi_path_rc_rdma);
+    md->super.multi_path.dc_rdma  = UCT_IB_MLX5DV_GET(cmd_hca_cap, cap,
+                                                      multi_path_dc_rdma);
+    md->super.multi_path.xrc_rdma = UCT_IB_MLX5DV_GET(cmd_hca_cap, cap,
+                                                      multi_path_xrc_rdma);
+
+    if (cap_2 != NULL) {
+        md->super.multi_path.force = UCT_IB_MLX5DV_GET(cmd_hca_cap_2, cap_2,
+                                                       multi_path_force);
+    }
+
+    ucs_debug("%s: multi_path support: force=%d rc_rdma=%d dc_rdma=%d "
+              "rcx_rdma=%d",
```

Comment:
`xrc_rdma`, but  i think we do not need to maintain this value

Reply:
fixed